### ISSUE TITLE
WebSocketInboundEndpoint.xml is not updated when apim.ws.handler.class is modified in the deployment.toml

### ISSUE THREAD

**Issue by ragwap** [wso2/product-apim#12448]
### Description:
It was observed that the WebSocketInboundEndpoint.xml is not updated when following property is modified in the deployment.toml:

```
[apim.ws.handler]
class = "some.handler"
```

### Steps to reproduce:

- Add the below configuration to the deployment.toml of WSO2 API Manager 3.2.0.

```
[apim.ws.handler]
class = "com.some.handler"

[apim.wss.handler]
class = "com.some.handler"
```

- Start the server.
- View the <APIM_HOME>/repository/deployment/server/synapse-configs/default/inbound-endpoints/WebSocketInboundEndpoint.xml and <APIM_HOME>/repository/deployment/server/synapse-configs/default/inbound-endpoints/SecureWebSocketInboundEndpoint.xml files.
- You will notice that only the SecureWebSocketInboundEndpoint has been updated with the changes.

### Reason:

Only the **SecureWebSocketInboundEndpoint.xml.j2** is available under **<APIM-HOME>/repository/resources/conf/templates/repository/deployment/server/synapse-configs/default/inbound-endpoints** directory.

### Affected Product Version:

APIM-3.2.0

### Environment details (with versions):
- OS:
- Client:
- Env (Docker/K8s):

---

**Comment by wasuradananjith** [comment]
This is fixed for APIM 3.2.0 via [1] and 2], and for master via [3].

[1]  https://github.com/wso2-support/product-apim/pull/1201
[2] https://github.com/wso2-support/product-apim/pull/1183
[3] https://github.com/wso2/product-apim/pull/12468/

**Comment by nandika** [comment]
Closing as fix has been merged
